### PR TITLE
Update comm_norms.adoc

### PR DESCRIPTION
**Pull request by dickersonpaula**
@lhawthorn I really enjoyed this chapter. Thanks so much for writing it. 

Hopefully, my edits/suggestions make sense. Please reach out with any questions or concerns.

Thanks,

Paula

### CODE REVIEW

**Comment by semioticrobotic** [comment]
Nice technique using your initials to mark editorial comments, @dickersonpaula. This means that anyone going through the piece and looking for items to address can do so with a good ol' Ctrl+F. I recommend this practice to other @theopensourceway/editors too, if they haven't already developed their own preferences.

**Comment by dickersonpaula** [comment]
Bryan,
I can't take credit for the idea. It was actually yours. You suggested it
when you were walking me through GitHub.
Thanks,
Paula

Paula Dickerson

She/Her/Hers

Senior Editor, Internal communications

Red Hat <https://www.redhat.com/>

100 E. Davie Street

Raleigh, NC 27601

pdickers@redhat.com    T: 919-835-6630
@redhatjobs <https://twitter.com/redhatjobs>   redhatjobs
<https://www.facebook.com/redhatjobs> @redhatjobs
<https://instagram.com/redhatjobs>
<https://www.redhat.com/>

File: comm_norms.adoc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 == Introduction
 
-While the tools that projects use to communicate continually evolve, best practices for communicating amongst project participants rarely change.
+While the tools projects use to communicate continually evolve, best practices for communicating among project participants rarely change.
```
Comment (lhawthorn) [inline comm_norms.adoc:5]: This wording is better and the change should be merged, think it was.

File: comm_norms.adoc [hunk 2]
```diff
@@ -12,18 +12,30 @@ And we will conclude with some observations on the ways ever-changing tooling an
 As the number of free and open source software projects has increased over time, the communication tools projects use—both to connect with their users and to coordinate their developers and contributors—have evolved.
 Internet Relay Chat (IRC) was once the default choice for real-time project chat.
 Many large projects conducted development discussions solely via their electronic mailing lists, https://en.wikipedia.org/wiki/Freecode[Freshmeat] was the place to hear about new project releases or updates to existing projects, and discussions about feature design and project improvements occurred via email and were included in a project's mailing list archives.
+PD--The sentence in line 14 is quite long. Consider breaking into two sentences (most likely between "projects" and "discussion").
```
Comment (lhawthorn) [inline comm_norms.adoc:15]: I agree with this change.

File: comm_norms.adoc [hunk 3]
```diff
@@ -12,18 +12,30 @@ And we will conclude with some observations on the ways ever-changing tooling an
...
+PD--Admittedly, I'm not the most tech-savvy person you'll find, but I had a hard time following the sentence above. Can it be simplified?
+
+* While some communities still employ email for asynchronous communications, others have found they can achieve better results by offering their users the opportunity to collaborate in online forums, a format more familiar to many developers who began coding at the same time they were sharing news and perspectives on then-new and popular forum sites like Reddit and Imgur.
+Advocates of the use of forums find the visual layout more appealing, and forum administrators often prefer the medium's fine grained access controls easier to use for moderation. (For example, being able to stop all posts in a particular thread during a heated discussion is relatively easy via most forum software—not so with mailing lists.)
+PD--How would you feel about "Forum advocates" instead of "Advocates of the use of forums"? It's a little tighter. It is also parallel with "forum advocates" later in the sentence.
```
Comment (lhawthorn) [inline comm_norms.adoc:26]: Yes we should make this change.

File: comm_norms.adoc [hunk 4]
```diff
@@ -12,18 +12,30 @@ And we will conclude with some observations on the ways ever-changing tooling an
 As the number of free and open source software projects has increased over time, the communication tools projects use—both to connect with their users and to coordinate their developers and contributors—have evolved.
 Internet Relay Chat (IRC) was once the default choice for real-time project chat.
 Many large projects conducted development discussions solely via their electronic mailing lists, https://en.wikipedia.org/wiki/Freecode[Freshmeat] was the place to hear about new project releases or updates to existing projects, and discussions about feature design and project improvements occurred via email and were included in a project's mailing list archives.
+PD--The sentence in line 14 is quite long. Consider breaking into two sentences (most likely between "projects" and "discussion").
 
 Today, the landscape has shifted:
+PD--The bullets that follow are quite long. Consider either paring them down so they are the length of one to two relatively simple sentences or reformat into paragraphs.
```
Comment (lhawthorn) [inline comm_norms.adoc:18]: I am open to reformatting into paragraphs for a later revision of the text, but I am OK with them going live as they currently are to make the 1.0 release.

File: comm_norms.adoc [hunk 5]
```diff
@@ -12,18 +12,30 @@ And we will conclude with some observations on the ways ever-changing tooling an
...
+Advocates of the use of forums find the visual layout more appealing, and forum administrators often prefer the medium's fine grained access controls easier to use for moderation. (For example, being able to stop all posts in a particular thread during a heated discussion is relatively easy via most forum software—not so with mailing lists.)
+PD--How would you feel about "Forum advocates" instead of "Advocates of the use of forums"? It's a little tighter. It is also parallel with "forum advocates" later in the sentence.
+
 * While Freshmeat remains active (now renamed to Freecode), free and open source software projects have also made extensive use of social media services like Twitter and Facebook to interact with their user and developer communities.
 Obviously, social media serves as an excellent vehicle for project marketing and keeping audiences abreast of major project developments.
+PD--Consider deleting "Obviously" as not everyone in the audience might have that feeling.
```
Comment (lhawthorn) [inline comm_norms.adoc:30]: Yes, we should delete obviously.

File: comm_norms.adoc [hunk 6]
```diff
@@ -12,18 +12,30 @@ And we will conclude with some observations on the ways ever-changing tooling an
...
+PD--This sentence would be a good candidate for a bulleted list. Doing so would better ensure social media's capabilities aren't lost within the body of the paragraph. However, it would be better if you didn't have a list within a list, which gets back to the original comment about the bullets in this list being rather long.
+
 And many users now expect to receive basic technical support via these platforms.
 * While mailing list archives may no longer serve as a definitive record of project discussions, http://github.com/[GitHub]'s popularity as an online development and collaboration platform, has meant many projects now manage their development discussions via GitHub issues, using GitHub's "tags" feature to differentiate, for instance, between discussions addressing implementation of a new feature and those reporting that the software does not work as intended.
+PD--Would it be possible to break this sentence up into more manageable chunks for the reader?
```
Comment (lhawthorn) [inline comm_norms.adoc:37]: Tagging as a future to do for a later iteration of the chapter.

File: comm_norms.adoc [hunk 7]
```diff
@@ -32,26 +44,31 @@ And yet while the tools free and open source projects use to achieve communicati
 
 People must understand the "who," "what," "where," "when," "why," and "how" of working with an open source software community.
 Thoughtfully established and well-maintained communication channels serve the following functions for an open source project and its participants:
+PD--Consider making the teaser into the list more active by switching verbs to something along the lines of "empower," "enable," or even "allow."
```
Comment (lhawthorn) [inline comm_norms.adoc:47]: Tagging as a future to do for a later iteration of the chapter.

File: comm_norms.adoc [hunk 8]
```diff
@@ -32,26 +44,31 @@ And yet while the tools free and open source projects use to achieve communicati
...
 Understanding these audiences and prioritizing their unique needs can help you craft better materials.
 
 For example, consider the basic distinction between project _users_ and project _developers_.
-Someone interested in _using_ the software will be interested in things like how to install it, how to configure it, and what they will need to know to accomplish the task at hand (e.g., build a website).
+Someone interested in _using_ the software will be interested in things like how to install it, how to configure it, and what they need to know to accomplish the task at hand (e.g., build a website).
 Developers will also be interested in such details, but will need _additional_ detail if they are going to contribute effectively to the project.
 
 For example, if your project observes certain coding conventions, be sure to include those conventions in the developer documentation.
-Having a patch rejected by a maintainer because you haven't indented code a certain way can be rather demotivating! Users, however, likely won't need these details.
+PD--You've got two paragraphs that both begin with "For example." Consider omitting one instance or using a synonym for variety's sake.
```
Comment (lhawthorn) [inline comm_norms.adoc:69]: Delete the for example on the second paragraph, it doesn't help the text.

File: comm_norms.adoc [hunk 9]
```diff
@@ -86,11 +103,14 @@ ____
 
 In this description—just a single paragraph—we learn:
 
-* What Drupal is (a content management system) 
-* What a content management system is (a tool to build websites) 
-* Why Drupal is a compelling choice (it's easy to use, reliable, secure, and flexible)
+* What Drupal is (a content management system). 
+* What a content management system is (a tool to build websites). 
+* Why Drupal is a compelling choice (easy to use, reliable, secure, and flexible).
+PD--Made some edits here so the bullets would be parallel.
```
Comment (lhawthorn) [inline comm_norms.adoc:109]: Thank you!

File: comm_norms.adoc [hunk 10]
```diff
@@ -86,11 +103,14 @@ ____
...
 
 Let's take another example from a popular project, Kubernetes.
+PD--Consider a colon instead of a comma after "project." It's more emphatic.
```
Comment (lhawthorn) [inline comm_norms.adoc:112]: Works for me, let's make this change.

File: comm_norms.adoc [hunk 11]
```diff
@@ -101,16 +121,19 @@ ____
 
 In this description, we immediately learn:
 
-* What Kubernetes is (a system for working with containerized applications, including deployment, scaling and management) 
-* How Kubernetes is abbreviated (little details like this one immediately deepen the comfort level of new arrivals; no one is expected to already know an arcane acronym upon arrival)
-* Where Kubernetes was developed (Google is noted as the originator of this code base, establishing the project as focused on enterprise applications and providing confidence that the software is well designed and maintained)
-* Kubernetes is open source (a user can expect to use, run, modify, and share changes to the code base; any questions about barriers to entry due to licensing fees, procurement processes, etc., are dismissed)
-* The project values community engagement (one can expect that contributions of code, documentation, etc.
-are welcome and encouraged)
+* What Kubernetes is. (It's a system for working with containerized applications, including deployment, scaling and management.)
```
Comment (lhawthorn) [inline comm_norms.adoc:124]: All these changes are fantastic, thank you Paula.

File: comm_norms.adoc [hunk 12]
```diff
@@ -101,16 +121,19 @@ ____
...
+* How Kubernetes is abbreviated. (Little details like this one immediately deepen the comfort level of new arrivals; no one is expected to already know an arcane acronym upon arrival.)
+* Where Kubernetes was developed. (Google is noted as the originator of this code base, establishing the project as focused on enterprise applications and providing confidence that the software is well designed and maintained.)
+* Kubernetes is open source. (A user can expect to use, run, modify, and share changes to the code base; any questions about barriers to entry due to licensing fees, procurement processes, etc., are dismissed.)
+* The project values community engagement. (One can expect that contributions of code, documentation, etc.
+are welcome and encouraged.)
 
 === Getting started
 
-The processes for creating good "getting started" documentation—occasionally called "onboarding documentation"—are outside the scope of this chapter (see the dedicated "Onboarding" section in this guidebook for more detail).
+The processes for creating good "getting started" documentation—occasionally called "onboarding documentation"—are outside the scope of this chapter.
```
Comment (lhawthorn) [inline comm_norms.adoc:133]: Tagging as a future to do for a later iteration of the chapter.

File: comm_norms.adoc [hunk 13]
```diff
@@ -101,16 +121,19 @@ ____
...
+PD--A general observation here: There are several em dashes in this paragraph (and the chapter as a whole). I love em dashes. They are my favorite punctuation mark. However, when used too often, their effectiveness is diluted. Consider replacing them with other punctuation throughout the chapter.
```
Comment (lhawthorn) [inline comm_norms.adoc:134]: Tagging as a future to do for a later iteration of the chapter.

File: comm_norms.adoc [hunk 14]
```diff
@@ -168,61 +193,84 @@ It can even encourage adoption of open source alternatives to the vendor's produ
 
  === What is an issue tracker?
 
-An issue tracker (sometimes also known as a "bug tracker, "issues list," or "issue queue") is a tool that allows people to submit reports when they encounter instances where they believe the software is not working as intended.footnote:[The authors are grateful for the work of Kent C.Dodds and Sara Drasner in their article https://css-tricks.com/open-source-etiquette-guidebook/[An Open Source Etiquette Guidebook], accessed 24 June 2020 12:52 CET.] Some projects manage their entire development workflow via their issue trackers as a way to monitor pending tasks and allow for collaborative commenting and review of work in progress.
+An issue tracker (sometimes also known as a "bug tracker, "issues list," or "issue queue") is a tool that allows people to submit reports when they encounter instances where they believe the software is not working as intended.footnote:[The authors are grateful for the work of Kent C.Dodds a
```
Comment (lhawthorn) [inline comm_norms.adoc:196]: This changes appears to have deleted some text?

File: comm_norms.adoc [hunk 15]
```diff
@@ -168,61 +193,84 @@ It can even encourage adoption of open source alternatives to the vendor's produ
...
+Some projects manage their entire development workflow via their issue trackers as a way to monitor pending tasks and allow for collaborative commenting and review of work in progress.
+PD--You may want to flip flop the order of the sentence here and begin with "As a way to monitor ..." to give more emphasis to the motivation.
```
Comment (lhawthorn) [inline comm_norms.adoc:198]: Tagging as a future to do for a later iteration of the chapter.

File: comm_norms.adoc [hunk 16]
```diff
@@ -168,61 +193,84 @@ It can even encourage adoption of open source alternatives to the vendor's produ
...
+
 In this section, we'll discuss using an issue tracker for the purpose of reporting failures with the software.
-By reporting your issue using a project's issue tracker, you ensure that maintainers who are looking out for problems will see your report and act upon it.
+By reporting your issue using a project's issue tracker, you ensure maintainers who are looking out for problems see your report and act upon it.
 
 === Why file an issue?
 
 While filing an issue may seem more cumbersome than simply asking for help in real-time chat, it is important to do for several reasons:
+PD--Some of the bullets in this list are also quite long. Can you pare down the content so they're shorter? Or consider a different presentation (paragraphs/subheads)?
```
Comment (lhawthorn) [inline comm_norms.adoc:206]: Tagging as a future to do for a later iteration of the chapter.

File: comm_norms.adoc [hunk 17]
```diff
@@ -168,61 +193,84 @@ It can even encourage adoption of open source alternatives to the vendor's produ
...
 
 * Project contributors cannot keep track of all conversations occurring across various platforms, but they can always refer back to the issue tracker when they have an opportunity to work on improving the project.
-Real-time chat and social media are ephemeral communication channels; the issue tracker is a purpose built tool for recording and reviewing problems with the software.
-* Software projects often define their upcoming work plans by using their issue tracker as a key component—and perhaps their sole tool—to prioritize all possible areas to work on (simply put, the project's issue tracker is very often synonymous with the project's to-do list).
+PD-With an eye toward paring down content, maybe change to "... but they can always refer to the issue tracker to improve the project."?
```
Comment (lhawthorn) [inline comm_norms.adoc:209]: Tagging as a future to do for a later iteration of the chapter.

File: comm_norms.adoc [hunk 18]
```diff
@@ -168,61 +193,84 @@ It can even encourage adoption of open source alternatives to the vendor's produ
...
+(Simply put, the project's issue tracker is very often synonymous with the project's to-do list.)
 If your problem does not make its way into the issue tracker, it will likely not be addressed simply because a very busy person has forgotten the details of the problem.
-For this reason, you will often find that one of the first requests you receive when asking for help is to file an issue so that the project maintainers can keep track of the problem.
+For this reason, you will often find that one of the first requests you receive when asking for help is to file an issue so the project maintainers can keep track of the problem.
 * Filing an issue allows you and the project contributors to communicate asynchronously about the problem in an easy way, as all parties can refer back to and access the issue description and follow up comments at any time.
+PD--Be careful about using "easy" as what's easy for you may not be easy for your reader. It's subjective and can come across as an absolute. Try to soften by making it more relative.
```
Comment (lhawthorn) [inline comm_norms.adoc:218]: Tagging as a future to do for a later iteration of the chapter.

File: comm_norms.adoc [hunk 19]
```diff
@@ -168,61 +193,84 @@ It can even encourage adoption of open source alternatives to the vendor's produ
...
+
 * When you've uncovered a problem with the software, you might discover that the problem is actually the root cause of _another_ problem, or there may be a way in which _several_ problems are related.
 Issue tracking software allows project developers to easily group related issues together, which may aid in diagnosing a problem's root cause.
+PD--See comment above about "easy" and carry through with "easily."
```
Comment (lhawthorn) [inline comm_norms.adoc:222]: Tagging as a future to do for a later iteration of the chapter.

File: comm_norms.adoc [hunk 20]
```diff
@@ -168,61 +193,84 @@ It can even encourage adoption of open source alternatives to the vendor's produ
...
+
 * People often encounter the same issues with software, and many of them are filing issues with the project.
-Having multiple reports of the same problem can be very time consuming for the project maintainers, as they would then need to respond to each individual reporter about work in progress.
-Fortunately, issue trackers make this process easier by allowing maintainers to quickly and easily close issues by stating that they're duplicates of an existing one (and then asking the reporter to track work-in-progress in the "original" report).
+Having multiple reports of the same problem can be very time consuming for the project maintainers, as they then need to respond to each individual reporter about work in progress.
+Fortunately, issue trackers make this process easier by allowing maintainers to quickly and easily close issues by stating they're duplicates of an existing one (and then asking the reporter to track work-in-progress in the "original" report).
+PD--Additional instances of "easy" and "easily" to consider.
```
Comment (lhawthorn) [inline comm_norms.adoc:227]: Tagging as a future to do for a later iteration of the chapter.

File: comm_norms.adoc [hunk 21]
```diff
@@ -168,61 +193,84 @@ It can even encourage adoption of open source alternatives to the vendor's produ
...
+
 Project maintainers can then engage in broadcast-style communication to everyone experiencing the problem in one place, simplifying their work stream while still helping everyone who needs assistance.
+PD--"Simplifying" could be considered along the lines of "easy/easily." Maybe "streamlining" instead.
```
Comment (lhawthorn) [inline comm_norms.adoc:230]: Tagging as a future to do for a later iteration of the chapter.

File: comm_norms.adoc [hunk 22]
```diff
@@ -168,61 +193,84 @@ It can even encourage adoption of open source alternatives to the vendor's produ
...
 
 === Make finding it easy
+PD--Reference earlier comments about "easy." Also, does "it" refer to "issue"? If so, it may be too far from the antecedent.
 
 Make sure the location of your project's issue tracker is prominently displayed in your FAQ, as well as in your usage and development documentation.
 If people cannot figure out where to submit an issue, they will ask someone in the project where to do so—and supporting well-meaning users by offering repeated answers to very basic questions like this one can be quite time consuming.
-Do yourself and your community a favor and make your issue tracker very easy to find!
+PD--A longish sentence. Maybe break into multiple ones so it's easier for the reader.
```
Comment (lhawthorn) [inline comm_norms.adoc:237]: Tagging as a future to do for a later iteration of the chapter.

File: comm_norms.adoc [hunk 23]
```diff
@@ -168,61 +193,84 @@ It can even encourage adoption of open source alternatives to the vendor's produ
...
 
+Do yourself and your community a favor and make your issue tracker very easy to find!.
 === Use issue templates
 
 Not everyone who uses your software will be familiar with your community's conventions for filing a useful bug report.
 To save both you and the reporter time, offer an issue template to ensure you receive the information you need to reproduce the reported error and effectively triage it.
+PD--My non-technical background caused me to pause on "reporter" as I immediately thought of a journalist. Maybe recast the sentence to "To save everyone time" or "To save time" instead?
```
Comment (lhawthorn) [inline comm_norms.adoc:244]: Tagging as a future to do for a later iteration of the chapter.

File: comm_norms.adoc [hunk 24]
```diff
@@ -168,61 +193,84 @@ It can even encourage adoption of open source alternatives to the vendor's produ
...
+
 For example, you may need to know what version of the software was in use when an error occurred, or what operating system the user has on the computer running the software.
+PD--Maybe "... what version of the software or what operating system was being used" instead?
```
Comment (lhawthorn) [inline comm_norms.adoc:247]: Tagging as a future to do for a later iteration of the chapter.

File: comm_norms.adoc [hunk 25]
```diff
@@ -168,61 +193,84 @@ It can even encourage adoption of open source alternatives to the vendor's produ
...
 
-If you find yourself asking for the same information over and over again in response to different bug reports, then congratulations! You have uncovered an area of your template in need of improvement.
+If you find yourself asking for the same information over and over again in response to different bug reports, then congratulations. You have uncovered an area of your template in need of improvement.
 
 === Help wanted: Labeling issues for clarity and encouraging contribution
 
 Most modern issue trackers allow users to label issues they file, which can be useful for organizing project work.
 By differentiating between different types of requests—features for development, software errors, etc.—a project can be more organized and triage issues more efficiently.
+PD--Question the use of "project" as the subject for the second verb as a "project" can't technically "triage issues," but people can.
```
Comment (lhawthorn) [inline comm_norms.adoc:260]: Tagging as a future to do for a later iteration of the chapter.

File: comm_norms.adoc [hunk 26]
```diff
@@ -234,104 +282,138 @@ Remember to be gracious and thankful in your discussions with other project part
 === When issues become the subject of heated debate
 
 At times, the details of addressing a particular issue can cause tension or argument within the community.
+PD--"Argument" should either be made plural, or "an" should be inserted before it.
```
Comment (lhawthorn) [inline comm_norms.adoc:285]: Should be arguments, correct.

File: comm_norms.adoc [hunk 27]
```diff
@@ -234,104 +282,138 @@ Remember to be gracious and thankful in your discussions with other project part
...
+
 While healthy and respectful debate is part of any thriving project—software or otherwise—tempers can flare easily, and (as has been well documented) people tend to behave with less civility online than they would in person.
+PD--What's your source for "as has been well documented"?
```
Comment (lhawthorn) [inline comm_norms.adoc:288]: https://www.bbc.com/future/article/20180403-why-do-people-become-trolls-online-and-in-social-media

File: comm_norms.adoc [hunk 28]
```diff
@@ -234,104 +282,138 @@ Remember to be gracious and thankful in your discussions with other project part
...
 
-. Thank the people creating the software for their time and energy, especially if you are brand new to the project.
+. Thank the people creating the software for their time and energy, especially if you are new to the project.
 The individuals spending their (free) time creating free and open source software for you to use are also people who want to know their time is valued and their work appreciated.
+PD--Maybe "spare" instead of "free"?
+
 . Include as much information as you possibly can about the error you have encountered.
 If the project uses issue templates, fill it out as completely as possible.
-If you do not have the information requested or cannot determine how to get it yourself, then simply note what you have attempted to do in order to get the information.
+PD--There's an issue with number here. You've got templates (plural) and it (singular).
```
Comment (lhawthorn) [inline comm_norms.adoc:301]: Can we say "fill one out"?

File: comm_norms.adoc [hunk 29]
```diff
@@ -234,104 +282,138 @@ Remember to be gracious and thankful in your discussions with other project part
...
+
+If you do not have the information requested or cannot determine how to get it yourself, simply note what you have attempted to do in order to get the information.
 These details help mainters determine what they might need to do to assist you.
+PD--Did you mean "maintainers" here?
```
Comment (lhawthorn) [inline comm_norms.adoc:305]: I did, this should be fixed.

File: comm_norms.adoc [hunk 30]
```diff
@@ -234,104 +282,138 @@ Remember to be gracious and thankful in your discussions with other project part
...
+
 . If a project does not use issue templates, look at other issues that have been "closed‒fixed" or at merged pull requests to see how other people have filed bug reports.
+PD--Make sure the number for "templates" here agrees with whatever you decide in the bullet above.
+
 If the issue was fixed, chances are quite good that you'll be able to use these historical artifacts as examples of the sort of information necessary for reproducing an error.
 Replicate what you find in these reports and add more detail as you are able.
 
 === Quick tips for responding to issues well
+PD--Like some of the other bulleted lists, the bullets here are rather long. Can you pare down the content any more or try another presentation?
```
Comment (lhawthorn) [inline comm_norms.adoc:314]: https://www.bbc.com/future/article/20180403-why-do-people-become-trolls-online-and-in-social-media

Comment (lhawthorn) [inline comm_norms.adoc:314]: Will look at for future chapter.

File: comm_norms.adoc [hunk 31]
```diff
@@ -234,104 +282,138 @@ Remember to be gracious and thankful in your discussions with other project part
...
 
 ____
 "While the size and skill of the development community constrains the rate at which tickets can be resolved, the project should at least try to acknowledge each ticket the moment it appears.
 Even if the ticket lingers for a while, a response encourages the reporter to stay involved, because she feels that a human has registered what she has done (remember that filing a ticket usually involves more effort than, say, posting an email)."—Karl Fogel, Producing Open Source Software.footnote:[https://producingoss.com/en/producingoss-letter.pdf[_https://producingoss.com/en/producingoss-letter.pdf], page 64, accessed 24 June 2020 11:46 CET]
+PD--Would the quote here be better before the subhead?
```
Comment (lhawthorn) [inline comm_norms.adoc:319]: I have no thoughts on this and leave it to the book editors. We may have a style convention for such pull quotes.

File: comm_norms.adoc [hunk 32]
```diff
@@ -234,104 +282,138 @@ Remember to be gracious and thankful in your discussions with other project part
...
 
 === Having development discussions and other conversations in the issue tracker
 
 Conventional wisdom in the early days of open source software development held that communities should _not_ carry on development related discussions in the project's issue tracker.
-Project's instead preferred carrying on such conversations via mailing lists or in forums a number of reasons: people following the mailing list were able to comment and express their views and needs without needing to parse through the issue tracker, forum or mailing list conversations were seen as better for asynchronous and long-form communications, and popular issue trackers in the 1990s and early 2000s were an unwieldy way to engage in actual discourse.
-Last but not least, discovering why a particular technical decision was made when those details were buried in an issue tracker was difficult, especially since the issue would be in a "closed" state once the decision was made.
+Project's instead preferred carrying on such conversations via mailing lists or in forums a number of reasons:
```
Comment (lhawthorn) [inline comm_norms.adoc:343]: Will update in a future edition of the chapter.

File: comm_norms.adoc [hunk 33]
```diff
@@ -234,104 +282,138 @@ Remember to be gracious and thankful in your discussions with other project part
...
 Looking for a closed issue to explain the technical direction of the project was considered counterintuitive.
+PD--Final item in the list
 
 With the rise in popularity of GitHub as a one-stop platform for online development work, conversations in the project issue tracker have become mainstream.
 GitHub's issue system visually mirrors the typically expected visual interface for forum software, making discussions in its system seem natural for those who began their development careers when online forums were first gaining popularity.
 Further, time and resources necessary for maintaining a Mailman instance or additional forum software as part of project infrastructure became cumbersome when all other infrastructure could be managed via a single tool.
 The addition of features such as the ability to "+1" an issue, set fine-grained controls on notifications for specific issues, and lock specific issues for editing only by project maintainers (while still allowing others to view the issue) made the move to discussions in the issue tracker more palatable and effective.
+PD--Is it locking the issue to "editing only" or is it project maintainers should be the only editors?
```
Comment (lhawthorn) [inline comm_norms.adoc:360]: Project maintainers should be the only editors.

File: comm_norms.adoc [hunk 34]
```diff
@@ -234,104 +282,138 @@ Remember to be gracious and thankful in your discussions with other project part
...
 While excellent search capabilities in online issue trackers make finding closed issues easier, the flow of an issue discussion does not fulfill the same function as a narrative description of a particular implementation or an explanation of why a certain decision was made.
 Note, too, that some maintainers who are most intimately familiar with the project—those who can recall specific issue numbers for particular discussions with ease—will not always be available to help with the project work.
-Preserving the knowledge of key decisions in an easy to access way saves time for people later when they are working to uncover the why of project processes, saves time for maintainers so they need not rehash history regularly, and ensures that critical details on how and why decisions were made are always available even as project membership changes.
+Preserving the knowledge of key decisions in an easy-to-access way saves time for people working to uncover the why of project processes, saves time for maintainers so they need not rehash history regularly, and ensures that critical details on how and why decisions were made are always available even as project membership changes.
```
Comment (lhawthorn) [inline comm_norms.adoc:366]: Fix in future iteration.

File: comm_norms.adoc [hunk 35]
```diff
@@ -234,104 +282,138 @@ Remember to be gracious and thankful in your discussions with other project part
...
+PD--Should it be "about" instead of "of key..."? Also, this sentence is long. You could theoretically turn it into a bulleted list spotlighting each of the three things preserving knowledge allows for.
 
 TIP: If your project carries on most of its development discussions in the issue tracker, consider taking some small steps to highlight these discussions in other ways that will be most accessible and discoverable.
-For example, you may summarize the discussion of the issue in a blog, forum post, or project newsletter, thereby preserving cultural lore for the project at the same time as you more widely inform the community about the change.
-If the project does not maintain a blog or other publication mechanism that would be suitable for such a communication, consider adding a list of watershed issues to your project documentation so that newcomers can become quickly familiar with these critical topics, and for ease of reference for long-time project participants.
+PD--Accessible and discoverable to whom?
```
Comment (lhawthorn) [inline comm_norms.adoc:370]: I am not sure how to address this comment.

File: comm_norms.adoc [hunk 36]
```diff
@@ -234,104 +282,138 @@ Remember to be gracious and thankful in your discussions with other project part
...
 For users and contributors who are not native English speakers, this fact can raise significant barriers to participation.
-There are a few steps that projects can take to help those for whom English is not their first language to more effectively participate in the project.
+There are a few steps projects can take to help those for whom English is not their first language to more effectively participate in the project.
 
 === Prominently recognize community resources available in multiple languages
 
 Should your project be widely adopted and grow to the point that it hosts communication channels in more than one human language, make sure to list these resources prominently on your project's website.
 Include on the website a note that the project welcomes submissions from community members for resources that are not written in English, and when the project receives such submissions act promptly to get them included in your project documentation.
+PD--"your website" or "the project website" instead? You could also break into two sentences after "English."
```
Comment (lhawthorn) [inline comm_norms.adoc:387]: the project website is correct
Fine to break into two sentences.

File: comm_norms.adoc [hunk 37]
```diff
@@ -234,104 +282,138 @@ Remember to be gracious and thankful in your discussions with other project part
...
+
 As you would with any resource you point your community to, do your best to ensure the resource is helpful.
 If you are unable to vet the resources given current resources on the project, reference the fact that project maintainers have been unable to assess the resource themselves, and welcome feedback on its inclusion in the project's documentation.
```
Comment (lhawthorn) [inline comm_norms.adoc:390]: vet the resources given currently available person-hours,

File: comm_norms.adoc [hunk 38]
```diff
@@ -346,16 +428,18 @@ Maintainers should be explicit about their desire to recruit contributors focuse
...
 
 === Maintaining civil discourse
 
 As has been discussed throughout this chapter, maintaining kind and gracious communications is vital for the project's ongoing health and well-being.
-While assuming that everyone understands what "kind and gracious communication" looks like may seem natural, when dealing with a global audience one cannot assume a consistent meaning for all participants.
+While assuming everyone understands what "kind and gracious communication" looks like may seem natural, one cannot assume a consistent meaning for all participants, especially when dealing with a global audience.
 Project maintainers and community members do well to lead by example, but making an explicit statement about what constitutes civil discourse, what matters are off-topic for the project, and what is expected from anyone communicating with the project about matters that may cause conflict sets an appropriate tone for the project.
+PD--Consider breaking into two sentences to make the thought easier for the reader to digest. Also, is "about matters that may cause conflict" too restrictive. Shouldn't the expectations be for whenever you are communicating with the project?
```
Comment (lhawthorn) [inline comm_norms.adoc:442]: Fix in future edition of chapter.

File: comm_norms.adoc [hunk 39]
```diff
@@ -378,51 +462,54 @@ This excerpt from the Diversity Statement of the Dreamwidth project is an excell
...
+Otherwise people will know neither what is expected of them nor whether the project is a place where they will feel welcome and comfortable contributing their time and expertise.
 See this guidebook's chapter on governance for more information about codes of conduct.
 
 == Successfully communicating with an open source project
 
 So far, we've focused primarily on ways in which software project _maintainers_ can ensure the best possible outcomes for communication in their projects.
 However, contributors, too, can take a number of steps to ensure they're communicating effectively with their favorite open source communities.
-Here are just a few.
+Here are just a few:
+PD--Some of the bullets are rather long. Look to pare down when possible.
```
Comment (lhawthorn) [inline comm_norms.adoc:492]: fix in future iteration

File: comm_norms.adoc [hunk 40]
```diff
@@ -446,6 +533,8 @@ Though 20 or 30 years seems like ancient history on the internet, it is worth no
 Open source is still a blip (albeit a significant one) in a much longer technological timeline.
 
 Academic researchers have found open source software projects and their development methodologies particularly worthy of study, due to open source's disruptive influence in the software industry.
+PD--Consider flip flopping the order and beginning with "Due to open ...".
```
Comment (lhawthorn) [inline comm_norms.adoc:536]: I agree with this change

File: comm_norms.adoc (hunk 2 above)
Comment (quaid) [reply comm_norms.adoc:15]: Fixing this and any following items that remain to do by i) getting them into a single PR right now, and ii) resolving this conversation as a sign that I've made the change. I won't comment on each one from here out; a closing of a conversation means the same thing from me -- it's been checked as already done, or I just did it.

File: comm_norms.adoc (hunk 4 above)
Comment (quaid) [reply comm_norms.adoc:18]: Easy enough to drop the bullets this time. I'm going to add a few words to the lead-in paragraph to make it lead in better without it being bullets.

File: comm_norms.adoc [hunk 41]
```diff
@@ -12,18 +12,30 @@ And we will conclude with some observations on the ways ever-changing tooling an
...
+
 * While Freshmeat remains active (now renamed to Freecode), free and open source software projects have also made extensive use of social media services like Twitter and Facebook to interact with their user and developer communities.
 Obviously, social media serves as an excellent vehicle for project marketing and keeping audiences abreast of major project developments.
+PD--Consider deleting "Obviously" as not everyone in the audience might have that feeling.
+
 Social media also provides a vehicle for advertising upcoming events, live streaming of content or talks, generating interest in a project, and other forms of marketing and outreach that broadcast further than a project's newsletter, website, or blog.
+PD--This sentence would be a good candidate for a bulleted list. Doing so would better ensure social media's capabilities aren't lost within the body of the paragraph. However, it would be better if you didn't have a list within a list, which gets back to the original comment about the bullets in this list being rather long.
```
[the remark this answers is not in the document]
Comment (quaid) [reply comm_norms.adoc:33]: got it, once I removed the overall listing, this one fell in to place naturally; I think you'll like the look and flow, @dickersonpaula has a good eye/ear.

File: comm_norms.adoc (hunk 13 above)
Comment (quaid) [reply comm_norms.adoc:134]: I'm removing the comment inline and will start a separate style guide around this. We use emdashes quite a bit in other chapters, way way more than here, and that can be OK if we are consistent with how sparking/unsparingly we use them vs. commas, parenthesis, etc.

Instead we'll figure out how we want to use them editorially, then do some search function to find all them for manual adjustments if needed. I'm moving this action therefore out of this specific document and into the overall guide.

**Pull request merged**